oca_dependencies.txt

```diff
@@ -17,7 +17,7 @@ bank-payment
 queue
 partner-contact
 web https://github.com/daramousk/web 10.0-MIG-CKEditor
-report-print-send
+report-print-send https://github.com/nbornand/report-print-send 10.0
```

Comment:
CompassionCH instead of nbornand, because we merged your fix and we never know if you won't delete your repository after leavind :wink: